main.tf

```diff
@@ -1,6 +1,6 @@
 locals {
   db_subnet_group_name          = var.db_subnet_group_name != "" ? var.db_subnet_group_name : module.db_subnet_group.this_db_subnet_group_id
-  enable_create_db_subnet_group = var.db_subnet_group_name == "" ? var.create_db_subnet_group : false
```

Comment:
this is working wrongly, it is solving reported problem but breaking a execution when this is setup for `create_db_subnet_group=true`